src/main.c

```diff
@@ -433,13 +227,6 @@ enumerate_input_devices (Display *dpy)
 
                   xi_dev->is_ts = (vi->mode & DeviceMode) == Absolute;
                   xi_dev->dev = dev;
-#if 0
```

Comment:
Why is this removed?

Reply:
lets not litter the code base with this kind of #if 0 disabled debug print. therfore this ill formatted debug print needs to go. if you think its needed promote it to compiled in g_debug() and improve the output formating. if its not important enough to warrant that (imo it is not) then just gain this information via gdb when needed.